tests/server/amplitude.js

```diff
@@ -184,11 +200,31 @@ define([
       assert.equal(process.stderr.write.callCount, 1);
       const arg = JSON.parse(process.stderr.write.args[0]);
       assert.equal(arg.event_type, 'fxa_pref - disconnect_device');
+      assert.equal(arg.event_properties.reason, 'suspicious');
       assert.isUndefined(arg.device_id);
       assert.isUndefined(arg.event_properties.device_id);
       assert.isUndefined(arg.user_id);
     },
 
+    'settings.clients.disconnect.submit.duplicate': () => {
+      amplitude({
+        time: 'a',
+        type: 'settings.clients.disconnect.submit.duplicate'
+      }, {
+        headers: {}
+      }, {
+        deviceId: 'b',
+        flowBeginTime: 'c',
+        flowId: 'd',
+        lang: 'e',
+        uid: 'f'
+      });
+      assert.equal(process.stderr.write.callCount, 1);
+      const arg = JSON.parse(process.stderr.write.args[0]);
+      assert.equal(arg.event_type, 'fxa_pref - disconnect_device');
+      assert.equal(arg.event_properties.reason, 'duplicate');
```

Comment:
@philbooth Running this locally, I don't see the reason property in the console.

```
{"op":"amplitudeEvent","time":1508333300667,"device_id":"a6a2fb4ad984497db4f36428d676ae85","event_type":"fxa_pref - disconnect_device","session_id":1508333205463,"event_properties":{"device_id":"a6a2fb4ad984497db4f36428d676ae85","entrypoint":"menupanel"},"user_properties":{"flow_id":"9aac5b950cf41c3e216796d3fc9664781775eefeda51d8a8b7a33b6ffe68d890","ua_browser":"Firefox","ua_version":"56","experiments":["q3form_changes_cad_on_signin"]},"app_version":"97","language":"en","os_name":"Mac OS X","os_version":"10.12"}
```

I vaguely remember there being some differences when testing locally, would this be related?

Reply:
Huh, that's weird, it works for me here. Which reason are you choosing?

Reply:
Forgot to restart server >.< after switching branch...